fava/static/css/base.css

```diff
@@ -601,8 +602,16 @@ td .status-indicator {
       color: var(--color-background);
     }
 
-    & span {
+    & span::before {
```

Comment:
Why not just set the background color?

Reply:
This way the background overlaps a little bit to the right and to the left, making it easier on the eyes.

Reply:
Ah, I think that can be done more simply (and slightly better, since it will take the width of letters into account) by padding+negative margin.

Reply:
Can you incorporate this? I'm not quite sure how this works with negative margins…

Reply:
Just add a padding of .1em left and right (or however much you want it to overlap) and a margin of -.1em left and right

Reply:
Ahh, thanks for the hint! Will modify the code.

Reply:
@yagebu This does not work as well, because just setting the padding and margin does show that letters further down the word/sentence are on top of the z-scale, so the background overlaps the letter to the left of the highlighted section a bit, which looks weird. By working with `::before`, the background stays behind the text.